src/status_im/utils/gfycat/core.cljs

```diff
@@ -11,9 +11,12 @@
 
 (defn generate-gfy
   ([public-key]
-  (let [gen (rnd/rand-gen public-key)
-        first-adjective (pick-random gen adjectives/data)
-        second-adjective (pick-random gen adjectives/data)
-        animal (pick-random gen animals/data)]
-    (str first-adjective " " second-adjective " " animal)))
+  (case public-key
+    nil "Unknown"
+    "0" "Unknown"
```

Comment:
Why not extract the unknown name to a var? It's a special value.

Reply:
yes you're absolutely right !